src/main/java/net/minecraftforge/fml/client/SplashProgress.java

```diff
@@ -357,7 +357,11 @@ public void run()
                         clearGL();
                         setGL();
                     }
-                    Display.sync(100);
```

Comment:
This line isn't the problem. The problem is the _necessary_ Display.update call a few lines above.